packages/nova-lib/lib/server/render_context.js

```diff
@@ -31,23 +55,49 @@ const Context = function Context(loginToken) {
   }
 };
 
+export const withRenderContextRaw = (func, options = {}) => {
+  const newFunc = Meteor.bindEnvironment((req, res, next) => {
```

Comment:
So now every fiber is aware of both the `loginContext` that gives access to `this.userId` & `renderContext`. That's great!!